patches/@1inch+permit-signed-approvals-utils+1.4.10.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@1inch/permit-signed-approvals-utils/eip-2612-permit.utils.js b/node_modules/@1inch/permit-signed-approvals-utils/eip-2612-permit.utils.js
+index 3e41e2e..e2a7fee 100644
+--- a/node_modules/@1inch/permit-signed-approvals-utils/eip-2612-permit.utils.js
++++ b/node_modules/@1inch/permit-signed-approvals-utils/eip-2612-permit.utils.js
+@@ -161,7 +161,7 @@ class Eip2612PermitUtils {
+             .catch(() => this.getTokenNonceByMethod(tokenAddress, walletAddress, nonceMethodNameIndex + 1))
+             .then((res) => {
+             if (res === '0x' || Number.isNaN(Number(res))) {
+-                return Promise.reject(new Error('nonce is NaN'));
```

Comment:
In which cases `res` is not a valid number? Some specific tokens?

Reply:
That I don't know.
In the AAVE case specifically, it's returning `0x` instead of throwing, which is a problem because it doesn't check the other nonce methods.
To be honest, I don't know why it started to not throw either, as I assume before it did given that we were able to call the right method in the past.
OR the AAVE token implementation changed 🤷